axon-server/installation/docker-k8s/axon-server-ee.md

```diff
@@ -231,33 +284,30 @@ Starting Axon Server EE using the docker-compose command is depicted below.
 $ docker-compose up
 ```
 
-## Kubernetes
+**Note 1**
```

Comment:
Nit: The style of this note is different than the other parts of the RefGuide (which are up to date).
Surely a style preference, but what I've been doing throughout is the following:

```
> **{short description of the note as header}**
>
> {note text}
```